tests/e2e/api/general.test.ts

```diff
@@ -174,13 +174,13 @@ describe("general api", () => {
       expect(symbol).toEqual("APT");
 
       const payload2: InputViewFunctionData = {
-        function: "0x1::coin::is_account_registered",
+        function: "0x1::coin::decimals",
         typeArguments: ["0x1::aptos_coin::AptosCoin"],
-        functionArguments: ["0x1"],
+        functionArguments: [],
       };
 
-      const isRegistered = (await aptos.view<[boolean]>({ payload: payload2 }))[0];
```

Comment:
have we figured out why it returns true instead of false? might be a higher level problem?

Reply:
Nope, good point though, we can investigate separately.

Reply:
I'm guessing some other test transfers coin to that account, but bc the test order is not deterministic the result is different sometimes.